GetStream/stream-chat-react
# fix: ignore link encoding when coming from dutchie CH-63341

# Summary
This change just makes it so that links that come from dutchie menus aren't re-encoded. Previously if menu items were pasted in to a chat, they would be re-encoded and would fall back to a dispensary's menu home page.

# Testing Instructions
* For a dispensary that has messaging enabled, open up a new message
* Paste in a link such as `https://www.serenity-cannabis.com/menu?dtche%5Bproduct%5D=tweed-penelope-1g`
* When the link appears in the chat click on it and ensure that you are taken directly to the product.

# Clubhouse Story
[ch63341](https://app.clubhouse.io/dutchie/story/63341/menu-deeplinks-break-when-sending-through-customer-messaging-due-to-being-replaced-with-the-html-entity-25)
